src/pineko/cli/check.py

```diff
@@ -38,11 +39,14 @@ def sub_compatibility(grid_path, operator_path, xif):
         rich.print("[red]Error:[/]", e)
 
 
+ScaleValue = namedtuple("ScaleValue", ["descr", "check"])
+
+
 class Scale(Enum):
     """Auxiliary class to list the possible scale variations."""
 
-    REN = auto()
-    FACT = auto()
+    REN = ScaleValue(" renormalization scale variations", check.contains_ren)
+    FACT = ScaleValue(" factorization scale variations", check.contains_fact)
```

Comment:
The initial space is kind of weird: drop it, and just add it back in the place you are using it
```suggestion
    REN = ScaleValue("renormalization scale variations", check.contains_ren)
    FACT = ScaleValue("factorization scale variations", check.contains_fact)
```